apps/app.mento.org/package.json

```diff
@@ -27,7 +27,8 @@
     "big.js": "^6.2.1",
     "bignumber.js": "^9.1.1",
     "decimal.js-light": "^2.5.1",
-    "ethers": "^5.7.2",
+    "ethers": "^5.8.0",
```

Comment:
Have you looked into dropping `ethers` altogether? i think `viem` should be able to do pretty much everything `ethers` does and more

Reply:
I did but came to the conclusion that the mento sdk itself needs to drop ethers as well

Reply:
yup, my bad, forgot about that dependency!